styles/roblox/catppuccin.user.css

```diff
@@ -161,15 +154,10 @@
         border-color: @accent-color;
         color: @base;
     }
-      .btn-growth-md, .stack .server-player-count-gauge .gauge-inner-bar, .create-chat-container .select-friends-btns .select-friends-save, .btn-growth-md > div {
-          background-color: @accent-color !important;
-          border-color: @accent-color !important;
-          color: @crust;
-      }
       .btn-secondary-md{
           border-color: @accent-color;
       }
-      .btn-growth-md:hover, .btn-growth-md:active, .rbx-upgrade-now .btn-secondary-md:hover, .rbx-upgrade-now .btn-secondary-md:active, .btn-primary-lg:hover, .btn-primary-lg:active, .btn-primary-xs:hover, .btn-primary-xs:active, .btn-common-play-game-lg:active, .btn-common-play-game-lg:hover, .btn-growth-xs:hover, .btn-growth-xs:active, .btn-growth-sm:hover, .btn-growth-sm:active, .btn-growth-lg:active, .btn-growth-lg:hover, .btn-primary-md:focus, .btn-primary-md:hover, .rbx-game-server-item .btn-control-xs:hover, .btn-secondary-md:hover, .rbx-friends-game-server-item .btn-control-xs:hover, .create-chat-container .select-friends-btns .select-friends-save:hover, .btn-primary-lg[disabled],.btn-buy-md:hover, .btn-buy-md:active, .btn-buy-md:focus, .btn-primary-md[disabled], .btn-cta-xs[disabled], .btn-control-md:hover, .btn-control-md:active, .btn-control-md:focus, .btn-control-sm:hover, .see-more-resellers:focus, .btn-control-sm:focus,  .btn-control-sm:active, .btn-control-xs:active, .btn-control-xs:focus {
+      .btn-growth-md:hover, .btn-growth-md:active, .rbx-upgrade-now .btn-secondary-md:hover, .rbx-upgrade-now .btn-secondary-md:active, .btn-primary-lg:hover, .btn-primary-lg:active, .btn-primary-xs:hover, .btn-primary-xs:active, .btn-common-play-game-lg:active, .btn-common-play-game-lg:hover, .btn-growth-xs:hover, .btn-growth-xs:active, .btn-growth-sm:hover, .btn-growth-sm:active, .btn-growth-lg:active, .btn-growth-lg:hover, .btn-primary-md:focus, .btn-primary-md:hover, .rbx-game-server-item .btn-control-xs:hover, .btn-secondary-md:hover, .rbx-friends-game-server-item .btn-control-xs:hover, .create-chat-container .select-friends-btns .select-friends-save:hover, .btn-primary-lg[disabled],.btn-buy-md:hover, .btn-buy-md:active, .btn-buy-md:focus, .btn-primary-md[disabled], .btn-cta-xs[disabled], .btn-control-md:hover, .btn-control-md:active, .btn-control-md:focus, .btn-control-sm:hover, .see-more-resellers:focus, .btn-control-sm:focus,  .btn-control-sm:active, .avatar-card-btns .accept-friend:hover,.select-group .select-option:hover, .select-group .select-option:focus, .select-group .select-option:active, .btn-growth-md:focus {
           background-color: @accent-color !important;
           border-color: @accent-color !important;
           filter: brightness(83%);
```

Comment:
What is this filter needed for?

Reply:
I'm suspecting that there may be a better way of achieving it, but the intention of the ``filter: brightness(83%)`` is to create a hover effect when hovering over certain elements like the play button or anywhere else where a hover effect would be needed. The reason why it was done like this was because the accent colors aren't provided in different lighter/darker variants of each (excluding between theme options) and just having it be accent-color would mean not having a hover effect (which would be suboptimal in my opinion.)

Reply:
Use functions from https://lesscss.org/functions/#color-operations. I usually use `darken`, `lighten`, and `fade` for operating on colors.

Reply:
Interesting, didn't know those existed. Will change it with that (prefer it anyway since the ``filter: brightness(83%)`` also changed the color of the text alongside the background + border color.)